court/tests.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+from court.models import Court
+from decimal import Decimal
+from django.db.utils import IntegrityError
+
+
+@pytest.mark.django_db
+class TestCourtModel:
+
+    @pytest.fixture
+    def court(self):
+        court = Court.objects.create(courtID=1, x=Decimal('11'), y=Decimal('22'),
+                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+        return court
+
+    def test_validate_instance(self, court):
+        court_in_db = Court.objects.get(pk=court.pk)
+        assert court_in_db == court
+
+    def test_create_court_with_existing_id(self, court):
+        with pytest.raises(IntegrityError):
+            Court.objects.create(courtID=1, x=Decimal('11'), y=Decimal('22'),
+                                 city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+
+    def test_create_valid_court(self):
+        Court.objects.create(courtID=1, x=Decimal('11'), y=Decimal('22'),
+                             city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
```

Comment:
use `full_clean` here to have the instance validated. See the player tests for examples.

Reply:
Will be added in the next PR. Thanks.

Reply:
Next PR?

Reply:
I meant that it would be updated next time I'll amend commit updates. 
I'm confused when we want to add `.full_clean()` and when not. I've been googling about it a bit but just got more confused. Could you please explain it to me?